releasenotes/notes/0.16/prepare-0.16-07bda95c73a22819.yaml

```diff
@@ -0,0 +1,15 @@
+---
+prelude: >
+  This is a new Rustworkx release  with many bug fixes and new features to
+  the library. The highlights of this release are:
+
+    * Enhanced support for type checking with `mypy` and `pyright`
+    * Support for reading GraphML files that are compressed
+    * New dominance algorithms
+
+  This release uses the `Python Stable ABI <https://docs.python.org/3.9/c-api/stable.html>`__,
+  and will be compatible with all versions from Python 3.9 onwards. The published
```

Comment:
Do you want to mention that 3.8 is no longer supported with this release? Since 0.15.0 supported 3.8.

Reply:
The release note for #1280 explicitly mentions it is removed. Given that we say it works with Python 3.9+ and 
I think we'll be fine with the explicit node below telling 3.8 was removed